### PR TITLE
Update documentation

### LINKED ISSUE #246: noggin vs fas2 quick reference
It would be nice to have a small user oriented document about the differences between fas2 and noggin. for example, in fas2 groups have admins that can make users into sponsors and when using noggin such a change needs a ipa admin to make the change for you. 

This would be something we could point users that are used to fas2 to more easily move to noggin.

### PR DESCRIPTION
**Pull request by ryanlerch**
Resolves: #246 

Signed-off-by: Ryan Lerch <rlerch@redhat.com>

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -100,3 +100,22 @@
...
+
+def run_apidoc(_):
+    from sphinx.ext import apidoc
+
+    apidoc.main(
+        [
+            "-f",
+            "-o",
+            os.path.join(topdir, "docs", "_source"),
+            os.path.join(topdir, "noggin"),
```
Comment (abompard) [inline docs/conf.py:115]: We can safely exclude the tests, just add `noggin/tests` to the arguments list.

**Review by abompard — CHANGES_REQUESTED** [review]
While investigating why it was not producing documentation for the controllers, I realized that we indeed do *not* have a `noggin/controller/__init__.py` file. Incredible that it works at all. Please add it (empty is fine).

File: docs/conf.py [hunk 2]
```diff
@@ -100,3 +100,21 @@
     "issue": ("https://github.com/fedora-infra/noggin/issues/%s", "#"),
     "pr": ("https://github.com/fedora-infra/noggin/pull/%s", "PR#"),
 }
+
```
Comment (abompard) [inline docs/conf.py:103]: CI is unhappy because you need one more blank line here I think. Just run black on the file and you'll be all set

**Review by abompard — CHANGES_REQUESTED** [review]

File: docs/conf.py (hunk 1 above)
Comment (ryanlerch) [reply docs/conf.py:115]: not sure what you mean by this comment, can you clarify for me?

Comment (abompard) [reply docs/conf.py:115]: Sure. As it is now, you are generating the apidocs for the unit tests modules, and I don't think it's useful. If you add `noggin/tests` to the command line, it'll exclude them from apidoc generation.

Comment (ryanlerch) [reply docs/conf.py:115]: thanks for that!

fixed and updated!

File: docs/index.rst [hunk 3]
```diff
@@ -28,8 +24,22 @@ Here is what works so far:
 
 Migrating your application:
 
-If your application currently uses FAS, and you wish to migrate it to use Noggin, please take a
-look at FASJSON's documentation which can be found `here <https://fasjson.readthedocs.io/>`_.
+If your application currently uses communicates with the Fedora Account System (FAS2) 
+using it's readonly API, this functionality will be provided by `fasjson <https://github.com/fedora-infra/fasjson/>`_
```
Comment (abompard) [inline docs/index.rst]: Missing full-stop! Sorry! :-)

**Review by abompard — CHANGES_REQUESTED** [review]

**Review by abompard — APPROVED** [review]

**Pull request merged**